addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -301,8 +320,8 @@ public void scan(HttpMessage msg, String param, String value) {
                     param);
 
             // Check 1: Start detection for Windows patterns
-            // note that depending on the AttackLevel, the number of prefixes that we will try
-            // changes.
+            // note that depending on the AttackLevel, additional variants including a null byte
+            // will be tried.
```

Comment:
I think this comment could just be removed, it seems clear just looking at the code.